src/window/concept_frame.rs

```diff
@@ -804,18 +827,25 @@ fn request_for_location(
 fn draw_buttons(
     canvas: &mut Canvas,
     width: u32,
+    scale: u32,
     maximizable: bool,
     mouses: &[Location],
     theme: &Theme,
 ) {
+    let scale = scale as usize;
+
     // Draw seperator between header and window contents
-    let division_line = line::Line::new(
-        (0, (HEADER_SIZE - 1) as usize),
-        ((width) as usize, (HEADER_SIZE - 1) as usize),
-        theme.get_secondary_color(false),
-        true,
-    );
-    canvas.draw(&division_line);
+
+    for i in 1..=scale {
+        let y = HEADER_SIZE as usize * scale - i;
+        let division_line = line::Line::new(
+            (0, y),
+            (width as usize * scale, y),
+            theme.get_secondary_color(false),
+            false,
```

Comment:
antialias `true` was breaking things here, since it was impossible to draw a horizontal line with it ((0,y),(x,y)). it should be drawn as ((0,y),(x,y - 1)) before, so I've disabled it, since it doesn't make much sense for horizontal line anyway.